scripts/prepare_disk.sh

```diff
@@ -196,6 +231,17 @@ if [ ! -r "${PARTITION_TABLE_FILE}" ]; then
     exit 1
 fi
 
+if [ ! -b "${TARGET_DISK}" ]; then
+    echo "Error, block device '${TARGET_DISK}' does not exist."
+    exit 1
+fi
+
+resize_needed="$(is_resize_needed; echo "${?}")"
```

Comment:
also note, that I merely meant it as a suggestion that we should discuss. While this works, what happens if is_resize_needed produces output? then we'll have 'output + echo 0'. granted, for the if test it does not matter, as everything not 1, will fail and trigger a resize.

alternativly we can also force the output to be really just the exit value; but i think it's a little uglier?
```sh
if [ "${result}" -ne 0 ]; then
	echo "nok"
else
	echo "ok"
fi
```